wiki/Difficulties/osu!taiko/en.md

```diff
@@ -0,0 +1,10 @@
+# osu!taiko
+
+There are the five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!taiko [mapset](/wiki/mapset).
```

Comment:
ditto, dood.

`There are five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!taiko [mapset](/wiki/mapset).`